log.go

```diff
@@ -363,3 +363,37 @@ func (l *Logger) should(lvl Level) bool {
 	}
 	return true
 }
+
+//Detect if the bytes to be printed is Binary or not
+//May be more robust method is needed here ?
+func binaryFmt(p []byte) bool {
+	if p[0] > 0x7F {
+		return true
+	}
+	return false
+}
+
+//DecodeIfBinaryToString - converts a binary formatted log msg to a
+//JSON formatted String Log message - suitable for printing to Console/Syslog etc
+func DecodeIfBinaryToString(in []byte) string {
```

Comment:
I would not export those methods.

Reply:
I think zerolog needs to provide decode method apis. Since CBOR package is internal to zerolog, i thought zerolog can provide one.

Reply:
If we provide a decoder, to be useful it should take a reader and write into a struct instead converting to json. It would be thus better to expose the cbor package.

I don’t think it is the role of a log lib to do that. If there is not cbor lib available to decode cbor, let’s expose the cbor lib as a separate project.

Reply:
good point. no longer expose this api.